package/MDAnalysis/analysis/msd.py

```diff
@@ -181,6 +181,29 @@
 
 We have now computed a self-diffusivity!
 
+Combining Multiple Replicates
+--------------------------------
+It is common practice to combine replicates when calculating MSDs. An example
+of this is shown below using MSD1 and MSD2.
+
+.. code-block:: python
+
+    u1 = mda.Universe(RANDOM_WALK, RANDOM_WALK_TOPO)
```

Comment:
Arguments to universe are wrong way round, this was my bad from original docs.

Reply:
Should I fix the arguments to universe in `Computing an MSD`?

Reply:
Yeah that would be great :)